models/story.model.js

```diff
@@ -22,4 +24,35 @@ db.on('open',function () {
 //Return all databases
 exports.testDB=function(){
     return allDatabases;
+};
+
+let storySchema=new Schema({

```

Comment:
Should we create a separate file for the schema, the code organization into files and folders similar to what @SaurabhaJirgi has followed in his pull request - https://github.com/climatetree/backend-mongo/pull/8 ?